### PR TITLE
REPLAY-1478 SR - add capability to record Steppers

### PR DESCRIPTION
**Pull request by mariusc83**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

**Review by xgouchet — APPROVED** [review]

File: library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/MaskAllNumberPickerMapperTest.kt [hunk 1]
```diff
@@ -0,0 +1,655 @@
...
+        val expectedTopDividerWireframe = MobileSegment.Wireframe.ShapeWireframe(
+            id = fakeTopDividerId,
+            x = fakeExpectedTopDividerXPos,
+            y = fakeExpectedTopDividerYPos,
+            width = fakeExpectedDividerWidth,
+            height = fakeExpectedDividerHeight,
+            shapeStyle = MobileSegment.ShapeStyle(
+                backgroundColor = fakeExpectedSelectedLabelHtmlColor
+            )
+        )
```
Comment (0xnm) [inline library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/MaskAllNumberPickerMapperTest.kt:174]: since in all the tests here and in other file most of the properties set are the same, won't it be easier and shorter to introduce the methods which create these objects reading `fakeXXX` values, and then in the test, since it is still a `data` class, just use `.copy` call with setting properties specific for the test. Otherwise it is hard to follow, test setup is very long.

File: library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/NumberPickerMapper.kt [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+import com.datadog.android.sessionreplay.utils.ViewUtils
+
+@RequiresApi(Build.VERSION_CODES.Q)
+internal open class NumberPickerMapper(
+    stringUtils: StringUtils = StringUtils,
+    private val viewUtils: ViewUtils = ViewUtils,
+    private val uniqueIdentifierGenerator: UniqueIdentifierGenerator = UniqueIdentifierGenerator
+) : BaseWireframeMapper<NumberPicker, MobileSegment.Wireframe>(stringUtils, viewUtils) {
+
+    @SuppressWarnings("LongMethod")
```
[the remark this answers is not in the document]
Comment (0xnm) [reply library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/NumberPickerMapper.kt:26]: I think the number of objects will be the same, they are just created here now, and can be moved to the dedicated functions. Anyway, if there is a concern, it is possible to help compiler with `inline`.

[the remark this answers is not in the document]
Comment (mariusc83) [reply library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/NumberPickerMapper.kt:26]: yeah, sometimes you need to create more and I was using an `intArray` sometimes instead of creating data classes.
